### PR TITLE
Update more acronyms in the Spec document

### PR DESCRIPTION
**Pull request by pnicolucci**
Update more Acronyms in the specification document.

### CODE REVIEW

File: spec/src/main/asciidoc/jakarta-stl.adoc [hunk 1]
```diff
@@ -1024,7 +1024,7 @@ _With a body_
 ....
 
 .*Body Content*
-JSP. The JSP container processes the body
+JSP. The Jakarta Server Pages container processes the body
 content, then the action trims it and processes it further.
```
Comment (pnicolucci) [inline spec/src/main/asciidoc/jakarta-stl.adoc:1028]: I didn't update this obvious one as I'm not quite sure how we should word it so left it for another day.

File: spec/src/main/asciidoc/jakarta-stl.adoc [hunk 2]
```diff
@@ -1873,12 +1873,12 @@ doing this.
 
 ==== Deferred Values
 
-As of JSP 2.1, the new unified Expression
+The unified Expression
 Language supports the concept of deferred expressions (using the #{}
```
Comment (pnicolucci) [inline spec/src/main/asciidoc/jakarta-stl.adoc:1877]: Removed the old specification reference, I think it reads fine how I have it now but open to your opinion.

**Comment by pnicolucci** [comment]
@arjantijms when you get a few minutes can you take a look and review these changes? Thanks in advance!!

File: spec/src/main/asciidoc/jakarta-stl.adoc (hunk 2 above)
Comment (arjantijms) [reply spec/src/main/asciidoc/jakarta-stl.adoc:1877]: Seems absolutely fine. I think most specs (at least the ones I'm looking at) did it that way.

**Comment by arjantijms** [comment]
> @arjantijms when you get a few minutes can you take a look and review these changes? Thanks in advance!!

I missed this one earlier, my apologies. I've reviewed now and it looks good!

**Pull request merged**